### PR TITLE
Add Configuration as a dependecy for ReadoutCard

### PR DESCRIPTION
**Pull request by kostorr**

### CODE REVIEW

**Comment by Barthelemy** [comment]
@ktf Could you refresh my memory about the failing travis test ?

**Comment by ktf** [comment]
you should remove the line with DYLD... which is not needed anymore.

File: readoutcard.sh [hunk 1]
```diff
@@ -32,6 +33,7 @@ cmake $SOURCEDIR                                                      \
       -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                             \
       ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}                      \
       ${COMMON_O2_VERSION:+-DCommon_ROOT=$COMMON_O2_ROOT}             \
+      ${CONFIGURATION_VERSION:+-DConfiguration_ROOT=$CONFIGURATION_ROOT} \
```
Comment (ktf) [inline readoutcard.sh]: Can configuration be optional or coming from the system? If not I would simply drop the `${CONFIGURATION_VERSION:+` part.

**Comment by ktf** [comment]
Merging this. Consider making Configuration mandatory and remove the support for optional presence, I guess.

**Pull request merged**